common/src/main/java/tachyon/util/network/NetworkAddressUtils.java

```diff
@@ -241,10 +338,164 @@ public static ServerSocket getSocket(final TServerSocket thriftSocket) {
    * @return the InetSocketAddress of the master
    */
   public static InetSocketAddress getMasterAddress(TachyonConf conf) {
-    String masterHostname =
-        conf.get(Constants.MASTER_HOSTNAME, getLocalHostName(conf));
+    String masterHost = conf.get(Constants.MASTER_HOSTNAME, getLocalHostName(conf));
     int masterPort = conf.getInt(Constants.MASTER_PORT, Constants.DEFAULT_MASTER_PORT);
-    return new InetSocketAddress(masterHostname, masterPort);
+    return new InetSocketAddress(masterHost, masterPort);
+  }
+
+  /**
+   * Helper method to get the {@link InetSocketAddress} connect address of the master.
```

Comment:
getMasterAddress() is old method. getMasterConnectAddress() is new method I added. Actually, we only need one method to retrieve the master connect address (in the case of using wildcard as binding address).